src/coreclr/src/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -4085,24 +4085,6 @@ internal static object CreateEnum(RuntimeType enumType, long value)
             bool[]? byrefModifiers, int culture, string[]? namedParameters);
 #endif // FEATURE_COMINTEROP
 
-#if FEATURE_COMINTEROP_UNMANAGED_ACTIVATION
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        internal static extern Type? GetTypeFromProgIDImpl(string progID, string? server, bool throwOnError);
-
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        internal static extern Type? GetTypeFromCLSIDImpl(Guid clsid, string? server, bool throwOnError);
-#else // FEATURE_COMINTEROP_UNMANAGED_ACTIVATION
-        internal static Type GetTypeFromProgIDImpl(string progID, string? server, bool throwOnError)
-        {
-            throw new NotImplementedException("CoreCLR_REMOVED -- Unmanaged activation removed");
```

Comment:
Calling `Type.GetTypeFromProgID` on Unix gives this today:
```
Unhandled exception. System.NotImplementedException: CoreCLR_REMOVED -- Unmanaged activation removed
   at System.RuntimeType.GetTypeFromProgIDImpl(String progID, String server, Boolean throwOnError)
   at System.Type.GetTypeFromProgID(String progID)
```
This change is changing it to nice COM interop not support PlatformNotSupportedException.